docs/security/trust/content_trust.md

```diff
@@ -177,31 +192,32 @@ choose a long, complex passphrase and be careful to keep the password and the
 key file itself secure and backed up. It is highly recommended that you use a
 password manager to generate the passphrase and keep it safe. There will be no
 way to recover this key. You can find the key in your config directory.
-Enter passphrase for new offline key with id a1d96fb: 
-Repeat passphrase for new offline key with id a1d96fb: 
-Enter passphrase for new tagging key with id docker.io/docker/trusttest (3a932f1): 
-Repeat passphrase for new tagging key with id docker.io/docker/trusttest (3a932f1): 
+Enter passphrase for new offline key with id a1d96fb:
+Repeat passphrase for new offline key with id a1d96fb:
+Enter passphrase for new tagging key with id docker.io/docker/trusttest (3a932f1):
+Repeat passphrase for new tagging key with id docker.io/docker/trusttest (3a932f1):
 Finished initializing "docker.io/docker/trusttest"
 ```
-When you push your first tagged image with content trust enabled, the  `docker` client
-recognizes this is your first push and:
- 
+When you push your first tagged image with content trust enabled, the  `docker`
+client recognizes this is your first push and:
+
  - alerts you that it will create a new offline key
  - requests a passphrase for the key
  - generates an offline key in the `~/.docker/trust` directory
  - generates a tagging key for in the `~/.docker/trust` directory
- 
-The passphrase you chose for both the offline key and your content key-pair should
-be randomly generated and stored in a *password manager*. 
 
-It is important to note, if you had left off the `latest` tag, content trust is skipped.
-This is true even if content trust is enabled and even if this is your first push. 
+The passphrase you chose for both the offline key and your content key-pair
+should be randomly generated and stored in a *password manager*.
+
+It is important to note, if you had left off the `latest` tag, content trust is
```

Comment:
Not something that will be fixed right now, but it would be awesome to have some kind of standardized callout box to highlight things like this in the docs.